packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -282,7 +284,6 @@ class WebAssetServer implements AssetReader {
       useSseForDebugProxy: useSseForDebugProxy,
       useSseForDebugBackend: useSseForDebugBackend,
       useSseForInjectedClient: useSseForInjectedClient,
-      serveDevTools: false,
```

Comment:
This change was required to roll in dwds, was the fix for the removal of the devtools dep I landed to resolve the version conflict post mortem